hledger-lib/Hledger/Data/Amount.hs

```diff
@@ -259,55 +260,43 @@ multiplyAmountAndPrice n a@Amount{aquantity=q,aprice=p} = a{aquantity=q*n, apric
 isNegativeAmount :: Amount -> Bool
 isNegativeAmount Amount{aquantity=q} = q < 0
 
+-- | Round an Amount to its specified precision
```

Comment:
Round an Amount's Quantity to its specified display precision. If the specified precision is NaturalPrecision, this is a no-op.